mypy/checkexpr.py

```diff
@@ -1061,15 +1061,54 @@ def check_op(self, method: str, base_type: Type, arg: Node,
             # method of the other operand instead.
             rmethod = self.get_reverse_op_method(method)
             arg_type = self.accept(arg)
+            base_arg_node = TempNode(base_type)
+            error_results = []  # type: List[Tuple[Tuple[Type, Type], MessageBuilder]]
```

Comment:
Hm... The logic involving this variable is confusing. If you're just going to report the first error, then why have a list? And should we really report the first error, not the last one?

(I do understand some of the logic -- if there's an error you still have to try other things which may succeed. But it's a lot of code.)

(Also, you can probably save on line breaks by setting your limit to 99 chars, which is what we use -- we've given up on 80 for mypy.)